### PR TITLE
docs(zh): add guide to install istio in tencent cloud

### PR DESCRIPTION
**Pull request by Xunzhuo**
Signed-off-by: Xunzhuo <mixdeers@gmail.com>

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by Xunzhuo** [comment]
PTAL @kebe7jun

File: content/zh/docs/setup/platform-setup/tencent-cloud-mesh/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+title: 腾讯云
+description: 在腾讯云上快速创建 Istio 服务。
+weight: 65
+skip_seealso: true
+keywords: [platform-setup,tencent-cloud-mesh,tcm,tencent-cloud,tencentcloud]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+## 准备工作
```
Comment (kebe7jun) [inline content/zh/docs/setup/platform-setup/tencent-cloud-mesh/index.md:11]: 差 ID。

File: content/zh/docs/setup/platform-setup/tencent-cloud-mesh/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+1. 选择相关的 **Tencent Kubernetes Engine** 或者 **Elastic Kubernetes Service** 集群。
+
+1. 选择在指定 Namespaces 下开启 Sidecar 自动注入。
+
+1. 配置外部请求绕过 Sidecar 直接访问的 IP 地址块，外部请求流量将无法使用 Istio 流量管理、可观测性等特性。默认所有外部请求转发至 Sidecar。
+
+1. 选择开启 Sidecar 就绪保障。
+
+    {{< tip >}}
+    开启后业务容器将等待 Sidecar ready 后再启动，将一定程度增加 Pod 启动时长，建议对于业务逻辑中有 Sidecar 功能强依赖的服务开启。
```
Comment (kebe7jun) [inline content/zh/docs/setup/platform-setup/tencent-cloud-mesh/index.md:56]:
```suggestion
    开启后业务容器将等待 Sidecar 就绪后再启动，将一定程度增加 Pod 启动时长，建议对于业务逻辑中有 Sidecar 功能强依赖的服务开启。
```

**Review by kebe7jun — APPROVED** [review]

**Comment by Xunzhuo** [comment]
Thanks @kebe7jun 👍

**Pull request merged**